composer.json

```diff
@@ -67,7 +68,8 @@
         "allow-plugins": {
             "composer/installers": true,
             "silverstripe/recipe-plugin": true,
-            "silverstripe/vendor-plugin": true
+            "silverstripe/vendor-plugin": true,
+            "dealerdirect/phpcodesniffer-composer-installer": true
```

Comment:
Required for composer to install correctly in CI with the new silverstripe/documentation-lint being added